.local.env

```diff
@@ -1,9 +1,17 @@
 NODE_ENV=development
 
+# OIDC config
+AUTH_ISSUER=https://auth.cube-creator.lndo.site
+AUTH_AUDIENCE=cube-creator
+AUTH_JWKS_URI=http://dex:5556/keys
```

Comment:
The `lndo.site` URLs cannot be used inside the containers right (which I think is a bug, reported as https://github.com/lando/lando/issues/2579)

Instead the service name has to be used to call dex from API to verify token